### DISCUSSION TITLE
Support ImmutableList

### DISCUSSION THREAD

**Discussion by mkowol-n** [mcarleio/konvert#51] [Ideas]
Hi
Are you planning to add converting lists to ImmutableList from org.jetbrains.kotlinx? 
It may help creating Stable Models for Jetpack Compose.
Also for now is there some easy workaround for this case?

**Comment by mcarleio** [comment]
I will have a look, how to implement stuff like this

**Reply by mcarleio** [reply]
Ok, with #57 I refactored the map and iterable converters to be able to reuse them like most other converters. Now it will be easier to extend Konvert to support further iterables and maps.
With #58 the support for kotlinx.collections.immutable is added.

**Reply by mkowol-n** [reply]
Ok, thanks. I didn't have time today to check it (just came back home) but i will try to find some time to check how it works tomorrow.
Anyway, thanks for this feature. This will help me a lot! :D

**Reply by mkowol-n** [reply]
Once again really thank you for your work. Now just the question is: When the new version with these changes will be released :D

**Reply by mcarleio** [reply]
Sorry, I was really busy last weeks... I did not release it right away because I have some open changes regarding https://github.com/mcarleio/konvert/issues/54 which I initially wanted to finish before releasing a new version.

But I just triggered a new release, so best case tomorrow a new version should be available!
